workspaces/adr/plugins/adr/src/alpha.tsx

```diff
@@ -50,8 +58,45 @@ export const adrSearchResultListItemExtension =
     },
   });
 
+/** @alpha */
+export const adrEntityContentExtension = EntityContentBlueprint.make({
+  name: 'entity',
+  params: {
+    defaultPath: '/adrs',
+    defaultTitle: 'ADRs',
```

Comment:
Should we add filters for Kind here? Thinking like this:

https://github.com/backstage/community-plugins/blob/a3f6805d4e13c280b56b9e48cb8e8384666429dc/workspaces/github-actions/plugins/github-actions/src/alpha/entityCards.tsx#L10